spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -33,14 +34,20 @@
 class SpyderIPy3Lexer(IPython3Lexer):
     # Detect !cmd command and highlight them
     tokens = IPython3Lexer.tokens
-    tokens['root'].insert(
-        0, (r'(!)(\w+)(.*\n)', bygroups(Operator, Keyword, Text)))
+    spyder_tokens = [
```

Comment:
Why is this change necessary?

Reply:
This adds syntax highlighting for magics which is nice for example:

`%timeit print("a", 0)`

would be highlighted and not left blank